doc/luasnip.txt

```diff
@@ -87,7 +87,7 @@ TEXTNODE                                                       *luasnip-text-nod
 The most simple kind of node; just text.
 >
 	s({trig='trigger'}, {
-		t({"Wow! Text!"}),
+		t"Wow! Text!",
```

Comment:
I think this could confuse newcomers to lua, especially if the ( ) are removed in only one example.

Reply:
Kinda? We see `require'foo`' everywhere and I think it's nice to remind people that this is possible, it almost looks like a DSL there.
I'll add it back if you disagree though.

Reply:
It looks nice, but I think to just have it in one example could lead newcomers to think that it is a mistake. Doing it consistently would be best, but that's something I could do as well.

Reply:
Fyi the formatter used by this project, stylua, will format `require'foo'` as `require('foo')` by default if I'm not mistaken. I suppose it'd be better to keep the formatting consistent between the code itself and the examples especially when trying to understand the plugin better by reading the code and comparing it to the examples.